pkg/server/grpc/tracee.go

```diff
@@ -470,92 +470,92 @@ var EventTranslationTable = [events.MaxBuiltinID]pb.EventId{
+	events.SecurityInodeMknod:           pb.EventId_security_inode_mknod,
+	events.SecurityPostReadFile:         pb.EventId_security_post_read_file,
+	events.SecurityInodeSymlinkEventId:  pb.EventId_security_inode_symlink_event_id,
+	events.SecurityMmapFile:             pb.EventId_security_mmap_file,
+	events.SecurityFileMprotect:         pb.EventId_security_file_mprotect,
+	events.SocketDup:                    pb.EventId_socket_dup,
+	events.HiddenInodes:                 pb.EventId_hidden_inodes,
+	events.KernelWrite:                  pb.EventId_kernel_write,
+	events.ProcCreate:                   pb.EventId_proc_create,
+	events.KprobeAttach:                 pb.EventId_kprobe_attach,
+	events.CallUsermodeHelper:           pb.EventId_call_usermode_helper,
+	events.DirtyPipeSplice:              pb.EventId_dirty_pipe_splice,
+	events.DebugfsCreateFile:            pb.EventId_debugfs_create_file,
+	events.SyscallTableCheck:            pb.EventId_syscall_table_check,
+	events.DebugfsCreateDir:             pb.EventId_debugfs_create_dir,
+	events.DeviceAdd:                    pb.EventId_device_add,
+	events.RegisterChrdev:               pb.EventId_register_chrdev,
+	events.SharedObjectLoaded:           pb.EventId_shared_object_loaded,
+	events.DoInitModule:                 pb.EventId_do_init_module,
+	events.SocketAccept:                 pb.EventId_socket_accept,
+	events.LoadElfPhdrs:                 pb.EventId_load_elf_phdrs,
+	events.HookedProcFops:               pb.EventId_hooked_proc_fops,
+	events.PrintNetSeqOps:               pb.EventId_print_net_seq_ops,
+	events.TaskRename:                   pb.EventId_task_rename,
+	events.SecurityInodeRename:          pb.EventId_security_inode_rename,
+	events.DoSigaction:                  pb.EventId_do_sigaction,
+	events.BpfAttach:                    pb.EventId_bpf_attach,
+	events.KallsymsLookupName:           pb.EventId_kallsyms_lookup_name,
+	events.DoMmap:                       pb.EventId_do_mmap,
+	events.PrintMemDump:                 pb.EventId_print_mem_dump,
+	events.VfsUtimes:                    pb.EventId_vfs_utimes,
+	events.DoTruncate:                   pb.EventId_do_truncate,
+	events.FileModification:             pb.EventId_file_modification,
+	events.InotifyWatch:                 pb.EventId_inotify_watch,
+	events.SecurityBpfProg:              pb.EventId_security_bpf_prog,
+	events.ProcessExecuteFailed:         pb.EventId_process_execute_failed,
+	events.SecurityPathNotify:           pb.EventId_security_path_notify,
+	events.SetFsPwd:                     pb.EventId_set_fs_pwd,
+	events.HiddenKernelModuleSeeker:     pb.EventId_hidden_kernel_module_seeker,
+	events.ModuleLoad:                   pb.EventId_module_load,
+	events.ModuleFree:                   pb.EventId_module_free,
+	events.ExecuteFinished:              pb.EventId_execute_finished,
+	events.ProcessExecuteFailedInternal: pb.EventId_security_bprm_creds_for_exec,
```

Comment:
I suppose that as `SecurityBprmCredsForExec` was removed as an event (remaining as a probe), this relation should be renamed as well.

Run `make -f builder/Makefile.protoc protoc-run` to update `event.pb.go`.

Insert in this PR the commit related to gprc bump (temporarily). So before merge it can be dropped and cherry-picked into a next PR. @rscampos know better about this proceedings.

Reply:
Ran that - it didn't seem to change tracee.go, rather api/v1beta1/event.pb.go